modules/main/src/index.js

```diff
@@ -67,7 +67,9 @@ export {
   // Effects
   _EffectManager,
   _Effect,
-  _ReflectionEffect
+  _ReflectionEffect,
+  LightingEffect,
+  PointLight
```

Comment:
Are we only supporting PointLight in this PR?

Reply:
No, we can support other source, but we can use luma.gl version for regular cases, so didn't extend them in deck.gl for now. Will extend them in deck.gl when there are more features.